mllib/src/test/scala/org/apache/spark/ml/tree/impl/RandomForestSuite.scala

```diff
@@ -161,6 +160,18 @@ class RandomForestSuite extends SparkFunSuite with MLlibTestSparkContext {
     }
   }
 
+  test("train with empty arrays") {
+    val lp = LabeledPoint(1.0, Vectors.dense(Array.empty[Double]))
+    val data = Array.fill(5)(lp)
+    val rdd = sc.parallelize(data)
+
+    val strategy = new OldStrategy(OldAlgo.Regression, Gini, maxDepth = 2,
+      maxBins = 5)
+    intercept[IllegalArgumentException] {
```

Comment:
It's good practice to include error messages using ```withClue``` to wrap ```intercept```.  Check out other test suites for examples.